UPDATING.md

```diff
@@ -0,0 +1,51 @@
+# Updating Quilted Fabric API
+
+Quilted Fabric API is a quite interesting project, because it isn't any project, it is simultaneously a fork of Fabric API, a repackaging of Fabric API, a collection of bridges that connect Fabric's APIs to QSL, and a mixture of both!
```

Comment:
> it is simultaneously a fork of Fabric API, a repackaging of Fabric API, a collection of bridges that connect Fabric's APIs to QSL, and a mixture of both!

shouldn't "a mixture of both" be a "mixture of the three" given there's 3 things listed here?

Reply:
I just copy-pasted this from 1.20.4, but I would agree. There is definitely a more meta PR update that is needed to improve the overall quality of QFAPI's .md files

Reply:
oh ok